internal/impl/openai/base_processor.go

```diff
@@ -26,7 +26,6 @@ func baseConfigFieldsWithModels(modelExamples ...any) []*service.ConfigField {
 	return []*service.ConfigField{
 		service.NewStringField(opFieldServerAddress).
 			Description("The Open API endpoint that the processor sends requests to. Update the default value to use another OpenAI compatible service.").
-			Advanced().
```

Comment:
Would it be OK to remove the Advanced tab where only the server_address is different from the common configuration, especially as the server_address is the first field listed in the docs?

Reply:
Seems fine to me!